internal/k8shandler/index_management_test.go

```diff
@@ -97,6 +97,13 @@ var _ = Describe("Index Management", func() {
 		BeforeEach(func() {
 			chatter = helpers.NewFakeElasticsearchChatter(
 				map[string]helpers.FakeElasticsearchResponses{
+					"_template/common.*,ocp-gen-*": {
```

Comment:
How about 'fmt.Sprintf("_template/common.*,%s-*", constants.OcpTemplatePrefix)'? Just to be consistent with line 80 in 'internal/elasticsearch/templates.go'